stores/amazon.py

```diff
@@ -1,22 +1,20 @@
 import json
-import secrets
+import os
 import time
 from os import path
-from price_parser import parse_price
 
 from amazoncaptcha import AmazonCaptcha
 from chromedriver_py import binary_path  # this will get you the path variable
 from furl import furl
+from price_parser import parse_price
 from selenium import webdriver
 from selenium.common.exceptions import NoSuchElementException
 from selenium.webdriver.common.keys import Keys
 from selenium.webdriver.support.ui import WebDriverWait
 
 from utils import selenium_utils
-from utils.json_utils import InvalidAutoBuyConfigException
```

Comment:
Correct me if I'm wrong, but removal here is dependent upon your other PR being merged.

Reply:
It does because of the import os dependency.  The imports can be cleaned up without the screen_shot change, but if you're willing, I'll enhance the save_screenshot function to be a non-breaking change for notification and then this can go in also.  The duplicate imports and unused imports keep getting in my way since I optimize them out of habit in my IDE.